content/docs/tasks/security/vault-ca/index.md

```diff
@@ -69,53 +66,74 @@ env:
   value: "istio_ca/sign/istio-pki-role"
 {{< /text >}}
 
-## Deploy a testing workload
+## Deploy testing workloads
 
-This section deploys a testing workload `httpbin`. When the sidecar of the
+This section deploys testing workloads `httpbin` and `sleep`. When the sidecar of a
 testing workload requests a certificate through SDS, Node Agent will send
 certificate signing requests to Vault.
 
-1.  Generate the deployment for an example `httpbin` backend:
+1.  Generate the deployment for an example `httpbin` backend and an example `sleep` backend:
 
     {{< text bash >}}
-    $ istioctl kube-inject -f @samples/httpbin/httpbin.yaml@ > httpbin-injected.yaml
+    $ istioctl kube-inject -f @samples/httpbin/httpbin-vault.yaml@ > httpbin-injected.yaml
+    $ istioctl kube-inject -f @samples/sleep/sleep-vault.yaml@ > sleep-injected.yaml
     {{< /text >}}
 
-1.  Deploy the example backend:
+1.  Create a service account `vault-citadel-sa`:
 
     {{< text bash >}}
-    $ kubectl apply -f httpbin-injected.yaml
+    $ kubectl create serviceaccount vault-citadel-sa
+    {{< /text >}}
+
+1.  Edit the service account `vault-citadel-sa` to use an example JWT token that has been configured
```

Comment:
Start with the conditional phrase to help readers determine upfront whether or not they want to proceed:
```suggestion
1.  Since the Vault CA requires the authentication and authorization of Kubernetes service accounts, you must edit the `vault-citadel-sa` service account to use the example JWT configured
```

Reply:
Done.